website/src/common/env-vars.ts

```diff
@@ -32,6 +32,11 @@ export const FLOATING_PREVIEW_BASE_URL: string = PRODUCTION_CONFIG
   : STAGING_CONFIG
   ? 'https://utopia.baby/'
   : BASE_URL
+export const PROPERTY_CONTROLS_INFO_BASE_URL: string = PRODUCTION_CONFIG
```

Comment:
Should this just be `export const PROPERTY_CONTROLS_INFO_BASE_URL: string = FLOATING_PREVIEW_BASE_URL`?

Reply:
I'm a little dubious of that just because someone changing it for the preview might accidentally bust the property controls.